### PR TITLE
Adding a test that checks remove in a workspace

### PR DESCRIPTION
**Pull request by yacinehmito**
I wanted to write a failing test for #4276. Turns out the test passes so the problem might not be related to the remove functionality itself. I'm still sharing the test though.

### CODE REVIEW

File: __tests__/commands/remove.js [hunk 1]
```diff
@@ -100,6 +100,14 @@ test.concurrent('removes a single scoped package', (): Promise<void> => {
   });
 });
 
+test.concurrent("removes a workspace's dependency", (): Promise<void> => {
+  return runRemove(['left-pad'], {}, 'workspaces-remove-workspace-dependency/workspace-child', async (config): Promise<
```
Comment (arcanis) [inline __tests__/commands/remove.js]: This test probably doesn't make what you want - the test framework will copy the folder referenced as third parameter, and by doing so will "forget" that "workspace-child" is a child of a "workspaces-remove-workspace-dependency".

Comment (yacinehmito) [reply __tests__/commands/remove.js]: Understood. How would you go about writing such a test?

Comment (arcanis) [reply __tests__/commands/remove.js]: Just use runInstall using the base fixture, then manually run remove (`new Remove(...)`) using the `{cwd: "..."}` option to execute it in the workspace subfolder.

**Comment by BYK** [comment]
@yacinehmito  - are you interested in updating this or shall we close the PR?

**Comment by yacinehmito** [comment]
Will do this weekend. If I don't before Monday feel free to close.

**Comment by BYK** [comment]
@yacinehmito - Closing due to inactivity. LMK if you're still interested.

**Pull request closed without merging**